robots/preservation/base.rb

```diff
@@ -8,10 +8,6 @@ class Base
 
     attr_reader :druid
 
-    def workflow_service
```

Comment:
Note, our base Class... er, Module already defined `attr_reader :workflow_service`.  Since we initialize WFS in `boot.rb` the default now does what we want.